pkgs/top-level/all-packages.nix

```diff
@@ -20817,4 +20817,6 @@ with pkgs;
   safeDiscardStringContext = callPackage ../build-support/safe-discard-string-context.nix { };
 
   simplehttp2server = callPackage ../servers/simplehttp2server { };
+
+  diceware = callPackage ../tools/security/diceware { };
```

Comment:
Please use `pythonPackages.callPackage`

Reply:
Sure, but can I ask why? So that I don't make the same mistake in the future :)

Reply:
The difference is that you don't have to use `with pythonPackages`, so it's less error-prone.